Waiter.py

```diff
@@ -0,0 +1,104 @@
+import os
+import re
+import time
+from colorama import Fore, Back, Style
+import pandas as pd
+from tabulate import tabulate as tb
+import numpy as np
+
+
+
+#Menu Options
+print("[ORDER & PAY HERE]")
+NumCust = input("How many of you will be dining with us today?")
```

Comment:
cast NumCust as int here. It allows the assumption that is made from the variable name, number of customers, to be held true throughout the code.